packages/devtools_shared/lib/src/extensions/extension_manager.dart

```diff
@@ -142,27 +137,17 @@ class ExtensionsManager {
   }) async {
     _assertUriFormat(rootFileUriString);
     final List<Extension> extensions;
-    try {
-      // TODO(https://github.com/flutter/devtools/issues/7944): this assumes
-      // that the .dart_tool/package_config.json file is in the package root,
-      // which may be an incorrect assumption for pub workspaces.
-      final packageConfigPath = path.posix.join(
-        rootFileUriString,
-        '.dart_tool',
-        'package_config.json',
-      );
-      extensions = await findExtensions(
-        'devtools',
-        packageConfig: Uri.parse(packageConfigPath),
-      );
-      logs.add(
-        'ExtensionsManager._addExtensionsForRoot find extensions for '
-        'config: $packageConfigPath, result: '
-        '${extensions.map((e) => e.package).toList()}',
-      );
-    } catch (e) {
-      rethrow;
-    }
+    final packageConfigPath = findPackageConfig(Uri.parse(rootFileUriString));
+    // TODO(sigurdm): Do we need to deduplicate package configs?
```

Comment:
we can remove this TODO. We deduplicate the detected extensions here: https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/extensions/extension_service.dart/#L203-L204

Reply:
Great, thanks!